docker-multinode/common.sh

```diff
@@ -123,40 +125,12 @@ kube::multinode::detect_lsb() {
   kube::log::status "Detected OS: ${lsb_dist}"
 }
 
-# Start a docker bootstrap for running etcd and flannel
-kube::multinode::bootstrap_daemon() {
```

Comment:
What about renaming common.sh to common-docker-bootstrap.sh and create a new common-cni.sh. We could choose the file by looking at the kubernetes version (@luxas has added such code already) 

It is not so bad, because the common-docker-bootstrap.sh will not be modified any more and only get bugfixes when needed. So, we do not really have a maintenance mess here. After a couple of months the old file is removed.